packages/json-schema/test/tuples.test.ts

```diff
@@ -0,0 +1,35 @@
+import { deepStrictEqual } from "assert";
+import { emitSchema } from "./utils.js";
+
+describe("jsonschema: tuples", () => {
+  it("emit tuples as items", async () => {
+    const schemas = await emitSchema(`
+      model Foo {
+        a: [string, int32]
+      }
+    `);
+    deepStrictEqual(schemas, {
+      "Foo.json": {
+        $id: "Foo.json",
+        $schema: "https://json-schema.org/draft/2020-12/schema",
+        type: "object",
+        required: ["a"],
+        properties: {
+          a: {
+            type: "array",
+            items: [
```

Comment:
I don't think this is correct, `items` is just an object with the array type.

Tuple is done with prefixItems i think 

https://json-schema.org/understanding-json-schema/reference/array#tupleValidation

Reply:
Interesting, what I found online said it was an array of schemas. However, that says it's for Draft 4 and 6, and we are on draft 2020-12, which is described here: https://json-schema.org/draft/2020-12/json-schema-core#section-10.3.1.1

It's not entirely clear to me what it should be... @bterlson what say you?

Reply:
Updated it to prefixItems

Reply:
The schemas for the tuple entries should be `prefixItems`. We had discussed and rejected adding any other length constraints.

Reply:
The length constraints just "happened" as a result of emitting the int type. Is there a way to suppress that for tuples or should it not be present on any int types?

Reply:
Min/max on tuple items is fine, was talking about limiting the length of the array schema. The output looks good!